.jenkins/Jenkinsfile-cryptography-wheel-builder

```diff
@@ -103,6 +121,29 @@ def build(version, label) {
                         otool -L `find .venv -name '_openssl*.so'` | grep -vG "libcrypto\\|libssl"
                     """
                 }
+            } else if (label.contains("docker")) {
+                linux32 = ""
+                if (imageName.contains("i686")) {
+                    linux32 = "linux32"
+                }
+                sh """#!/usr/bin/env bash
+                    set -x -e
+                    # Because we are doing this as root in the container, but we write to a mounted dir that is outside the container
+                    # we need to make sure we set these files writable such that the jenkins user can delete them afterwards
+                    mkdir -p tmpwheelhouse
+                    mkdir -p wheelhouse
+                    chmod -R 777 tmpwheelhouse
+                    chmod -R 777 wheelhouse
+
+                    $linux32 /opt/python/$version/bin/pip install cffi six idna asn1crypto ipaddress enum34
```

Comment:
Is there a better way for us to get the deps for cryptography without enumerating them here? We do a pip install later with --no-index so we need all the deps.